src/controller/main.cpp

```diff
@@ -66,6 +66,15 @@ DWORD WINAPI workerProcessThread(LPVOID lpParam)
 void process_command_line(int argc,char *argv[])
 {
 	std::string logfile = "nwnx_controller.txt";
+	LogLevel logLevel = LogLevel::info;
+
+	// Search for verbosity arg
+	for (int i = 1; i < argc; i++)
+	{
+		if (_stricmp(argv[i], "-verbose") == 0){
+			logLevel = LogLevel::trace;
+		}
+	}

```

Comment:
Just  a notice that if we want to handle the command line arguments, we might want to start using TCLAP or Boost libraries.

Reply:
yes, currently argument parsing is a mess and this should be reimplemented with a decent library.

However that would also mean breaking the current CLI interface, and diverging from nwn2server cli parsing (that expects a list of `-argument value`).